hack/verify-boilerplate.sh

```diff
@@ -18,7 +18,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-VERSION=v0.1.1
+VERSION=v0.1.3
```

Comment:
I guess this can be added to commit where we initially added the script?

Reply:
yes